.travis.yml

```diff
@@ -316,7 +316,8 @@ matrix:
       - gem update --system $(grep rubygems omnibus_overrides.rb | cut -d'"' -f2)
       - gem install bundler -v $(grep bundler omnibus_overrides.rb | cut -d'"' -f2)
       - sudo apt-get update
-      - sudo apt-get -y install squid3 git curl
+      - sudo apt-get -y install squid3 git curl apache2-utils
+      - sudo htpasswd -b -c /etc/squid3/passwords proxy proxy
```

Comment:
squid is bound to localhost so that proxy/proxy wont be guessable?  (although without the password we'd be creating an unauthenticated open proxy if we weren't binding to localhost, and this wouldn't make that any worse...)